### DISCUSSION TITLE
How to upload psd file?

### DISCUSSION THREAD

**Discussion by eduarddotgg** [vendurehq/vendure#1552] [Q&A]
I would like to upload psd file to my product but upload fails due to psd file mime type which is 'image/vnd.adobe.photoshop'. is there any way to prevent any image manipulation with it?

**Comment by michaelbromley** [comment]
Hi,

You should be able to enable this using the [`assetOptions.permittedFileTypes` config option](https://www.vendure.io/docs/typescript-api/assets/asset-options/#permittedfiletypes).

> is there any way to prevent any image manipulation with it?

I'm not sure what you mean by this part.

**Comment by eduarddotgg** [comment]
the issue is that psd has "image/.." whatever mime type and vendure (vendure's preview generator) tries to generate preview file and then just fails because it can't process psd file.

**Reply by michaelbromley** [reply]
aah ok I get it. I'll create an issue for this because at the very least, if the preview generations fails it should not cause the entire operation to fail.

**Reply by michaelbromley** [reply]
https://github.com/vendure-ecommerce/vendure/issues/1563

**Reply by eduarddotgg** [reply]
thanks!
